homeassistant/components/media_player/denonavr.py

```diff
@@ -55,27 +74,42 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     if cache is None:
         cache = hass.data[KEY_DENON_CACHE] = set()
 
+    # Get config option for show_all_sources
+    show_all_sources = config.get(CONF_SHOW_ALL_SOURCES) if not None else False
```

Comment:
This doesn't work ? also, there is already a default value specified for CONF_SHOW_ALL_SOURCES so also not necessary.